synth.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+common_templates = gcp.CommonTemplates()
+templates = common_templates.node_library()
+s.copy(templates, excludes=['README.md'])
+
+s.replace('.kokoro/test.bat', r'(call npm install \|\| goto :error)',
+r"""
+@rem Plugin integration tests on Windows are skipped, because there is no known
```

Comment:
Are we still running appveyor for this today? Want to make sure we aren't dropping coverage

Reply:
We aren't. AFAIK we don't report coverage from Windows (we didn't in the past).